homework.js

```diff
@@ -0,0 +1,136 @@
+function summation(number) {
+    let sum = 1;
+    for(let i=1;i<=number;i++){
+        sum+=i
```

Comment:
Nice job Modhi; but as I see here the summation will be ```summation(number) + 1``` because you set the initial value of ```sum``` equal to ```1``` so after the first iterative the ```sum``` will be equal to ```2``` rather than ```1```